internal/pkg/cli/svc_delete.go

```diff
@@ -131,6 +137,9 @@ func (o *deleteSvcOpts) Ask() error {
 		// When a customer provides a particular environment,
 		// we'll just delete the service from that environment -
 		// but keep it in the app.
+		if err := o.validateEnvName(); err != nil {
```

Comment:
Should we validate it outside if `if o.skipConfirmation`? 
Because the user could pass in an env but not want to skip confirmation.